app/views/decidim/action_delegator/_callout.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if Decidim::ActionDelegator::Delegation.granted_to?(current_user, consultation) %>
```

Comment:
We could probably add a helper here

Reply:
yeah, probably a helper like `#granted_to?` makes sense